cmd/ingester/app/flags.go

```diff
@@ -48,6 +48,24 @@ const (
 	SuffixParallelism = ".parallelism"
 	// SuffixHTTPPort is a suffix for the HTTP port
 	SuffixHTTPPort = ".http-port"
+	// SuffixAuthentication for enable or disable authentication when connect to kafka cluster.
+	SuffixAuthentication = ".authentication"
+	// KerberosPrefix for Kerberos configuration options
+	KerberosPrefix = ".kerberos"
+	// SuffixKerberosServiceName is the suffix for Kerberos service name
+	SuffixKerberosServiceName = ".serviceName"
+	// SuffixKerberosRealm is the suffix for Kerberos realm name
+	SuffixKerberosRealm = ".realm"
+	// SuffixKerberosUseKeyTab is the suffix determine if kerberos should use keytab file or password
+	SuffixKerberosUseKeyTab = ".useKeytab"
+	// SuffixKerberosUserName is Kerberos username
+	SuffixKerberosUserName = ".username"
+	// SuffixKerberosPassword is Kerberos password
+	SuffixKerberosPassword = ".password"
+	// SuffixKerberosConfig is path to the kerberos configuration file.
+	SuffixKerberosConfig = ".kerberosConfig"
```

Comment:
This option will already have a `.kerberos` prefix - so the option itself could just be named `.config`